src/v2/api/controllers/UserController.ts

```diff
@@ -87,12 +87,23 @@ export class UserController {
   @ApiBearerAuth()
   @ApiOkResponse()
   @ApiBadRequestResponse({
-    description: `Exceptions:\n
-                  InvalidEntityIdException: User with such id is not found
-                  AlreadyRegisteredException: User is already registered
-                  InvalidBodyException: Group id can not be empty
-                  InvalidBodyException: IsCaptain must be a boolean
-                  InvalidBodyException: IsCaptain can not be empty`,
+    description: `\n
+    InvalidEntityIdException:
+      User with such id is not found
+
+    AlreadyRegisteredException:
+      User is already registered
+
+    InvalidBodyException:
+      Group id can not be empty
+      IsCaptain must be a boolean
+      IsCaptain can not be empty
+    
+    AbsenceOfCaptainException:
+      Captain was not found
+    
+    AbsenceOfCaptainTelegramException:
+      Captain's telegramId was not found`,
   })
   @ApiForbiddenResponse({
     description: `NoPermissionException:\n
```

Comment:
Add one more ApiForbiddenResponse exeption to swagger 
![image](https://github.com/fictadvisor/fictadvisor-api/assets/116638814/bbbd813e-a7b9-46bc-9508-45120130d681)
![image](https://github.com/fictadvisor/fictadvisor-api/assets/116638814/c1f77676-854b-49f7-b97b-30089885def4)